### PR TITLE
Accessibility Checks

### PR DESCRIPTION
**Pull request by mmatongo**
Easy Checks Web Accessibility Report:

- Page titles - no issues found
- Image text alternatives - no issues found
- Text headings - see inline comment
- Button Color contrast - See inline comment
- Resize - no issues found
- Interaction - no issues found
- Moving content - no issues found
- Multimedia - no issues found
- The basic structure of the page - no issues found

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -203,15 +203,15 @@ header {
 }
 
 .frame {
-  color: #36b37e;
```
Comment (mmatongo) [inline css/styles.css:206]: Color does not pass the button contrast test

File: css/styles.css [hunk 2]
```diff
@@ -203,15 +203,15 @@ header {
...
+  color: #008756;
   font-weight: 600;
   font-size: 12px;
   line-height: 16px;
 }
 
 .text_btn {
   padding: 12px;
-  background-color: #36b37e;
```
Comment (mmatongo) [inline css/styles.css:214]: Color does not pass the button contrast test

File: css/styles.css [hunk 3]
```diff
@@ -259,7 +259,7 @@ h2 {
   font-size: 40px;
   font-weight: bold;
   line-height: 52px;
-  color: #36b37e;
```
Comment (mmatongo) [inline css/styles.css:262]: Color does not pass the button contrast test

File: css/styles.css [hunk 4]
```diff
@@ -447,7 +447,7 @@ form {
     line-height: 60px;
     text-align: center;
     letter-spacing: 0.37px;
-    color: #36b37e;
```
Comment (mmatongo) [inline css/styles.css:450]: Color does not pass the button contrast test, update color to make it it compliant.

**Review by lily-coder — COMMENTED** [review]
The form was lacking labels. So I added labels for accessibility.

**Review by hemant-soni-vst-au4 — CHANGES_REQUESTED** [review]
Hello Team, 

Good job so far! 👍 
You still need to work on some issues to go to the next project, but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

- [ ] There should be a detailed title of the page in the `<head>` tag. Like `Mark's portifolio`


Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lily-coder** [comment]
I have implemented the changes as requested by the reviewer by making the title of the page more descriptive

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**